packages/augur-ui/src/modules/market-cards/market-card.tsx

```diff
@@ -0,0 +1,215 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+
+import { CategoryTagTrail, MarketTypeLabel, MarketStatusLabel } from "modules/common/labels";
+import { OutcomeGroup, LabelValue, HoverIcon } from "modules/market-cards/common";
+import toggleTag from "modules/routes/helpers/toggle-tag";
+import toggleCategory from "modules/routes/helpers/toggle-category";
+import { MARKETS } from "modules/routes/constants/views";
+import makePath from "modules/routes/helpers/make-path";
+import MarketLink from "modules/market/components/market-link/market-link";
+import { CATEGORICAL, COPY_MARKET_ID, COPY_AUTHOR } from 'modules/common/constants';
+import { FavoritesButton } from "modules/common/buttons";
+import Clipboard from "clipboard";
+import { DotSelection } from "modules/common/selection";
+import { DateFormattedObject } from "modules/types";
+import { PaperClip, Person, MarketCreator, PositionIcon, DesignatedReporter, DisputeStake } from "modules/common/icons";
+import { MarketProgress } from "modules/common/progress";
+import ChevronFlip from "modules/common/chevron-flip";
+
+import Styles from "modules/market-cards/market-card.styles";
+
+interface MarketCardProps {
+  market: Object;
+  isLogged?: Boolean;
+  history: object;
+  location: object;
+  toggleFavorite: Function;
+  currentAugurTimestamp: number;
+  reportingWindowStatsEndTime: number;
+  condensed?: Boolean;
+  address: string;
+}
+
+interface MarketCardState {
+  expanded: Boolean;
+}
+
+export default class MarketCard extends React.Component<
+  MarketCardProps,
+  MarketCardState
+> {
+  state: FormDetailsState = {
+    expanded: false,
+  };
+
+  expand = () => {
+    this.setState({expanded: !this.state.expanded})
+  }
+
+  render() {
+    const {
+      market,
+      location,
+      history,
+      isLogged,
+      toggleFavorite,
+      currentAugurTimestamp,
+      reportingWindowStatsEndTime,
+      condensed,
+      address
+    } = this.props;
+
+    const s = this.state;
+
+    const {
+      description,
+      outcomesFormatted,
+      marketType,
+      scalarDenomination,
+      minPrice,
+      maxPrice,
+      categories,
+      id,
+      marketStatus,
+      isFavorite,
+      author,
+      reportingState,
+      endTime,
+      openInterestFormatted,
+      volumeFormatted,
+      tags
+    } = market;
+
+    const path =
+    location.pathname === makePath(MARKETS)
+      ? location
+      : { pathname: makePath(MARKETS) };
+
+    const process = (...arr) =>
+      arr.filter(Boolean).map(label => ({
+        label,
+        onClick: toggleCategory(label, path, history)
+      }));
+
+    const categoriesWithClick = process(categories[0]);
+    const tagsWithClick = tags.filter(Boolean).map(tag => ({
```

Comment:
tags don't exist anymore. They'll get removed from marketInfo in the near future.